webdriver/tests/bidi/browsing_context/reload/wait.py

```diff
@@ -0,0 +1,120 @@
+import asyncio
+import pytest
+
+pytestmark = pytest.mark.asyncio
+
+
+async def wait_for_reload(bidi_session, context, wait, expect_timeout):
+    # Ultimately, "interactive" and "complete" should support a timeout argument.
+    # See https://github.com/w3c/webdriver-bidi/issues/188.
+    if expect_timeout:
+        with pytest.raises(asyncio.TimeoutError):
+            await asyncio.wait_for(
+                asyncio.shield(
+                    bidi_session.browsing_context.reload(context=context,
+                                                         wait=wait)),
+                timeout=1,
+            )
+    else:
+        await bidi_session.browsing_context.reload(context=context, wait=wait)
+
+
+@pytest.mark.parametrize("wait", ["none", "interactive", "complete"])
```

Comment:
Can we verify the events order, like in e2e mapper tests?

Reply:
Done

Reply:
I added wait for event in a few places, and a check that they happened in the expected order. PTAL.